Essentials/src/main/java/com/earth2me/essentials/utils/NumberUtil.java

```diff
@@ -120,6 +120,15 @@ public static boolean isPositiveInt(final String sInt) {
         return Integer.parseInt(sInt) > 0;
     }
 
+    public static boolean isNumeric(final String sNum) {
```

Comment:
Why not `isInt`?

Reply:
`+1` will not throw an exception in `parseInt`